src/composables/transactionList.ts

```diff
@@ -0,0 +1,329 @@
+import {
+  AEX9_TRANSFER_EVENT,
+  fetchJson, getLocalStorageItem, setLocalStorageItem,
+  TX_DIRECTION,
+} from '../popup/utils';
+import { useAccounts } from './accounts';
+import JsonBig from '../lib/json-big';
+import { useAeSdk } from './aeSdk';
+
+const LOCAL_STORAGE_KEY = ['transactions_state'];
+
+const isInitalized = ref(false);
+
+const generateEmptyTransactionState = (): ITransactionsState => ({
+  loaded: [],
+  nextPageUrl: '',
+  pending: {},
+  tipWithdrawnTransactions: [],
+});
+const transactions = ref<ITransactionStateList>(
+  getLocalStorageItem(LOCAL_STORAGE_KEY) || {},
+);
+
+function getTransactionStateByAccountAddress(address: Encoded.AccountAddress) {
+  return transactions.value[address];
+}
+
+function getOrInitTransactionStateForAccount(address: Encoded.AccountAddress): ITransactionsState {
+  const accountTransactionState = getTransactionStateByAccountAddress(address);
+
+  if (accountTransactionState) {
+    return accountTransactionState;
+  }
+
+  transactions.value[address] = generateEmptyTransactionState();
+
+  return transactions.value[address];
+}
+
+function getTransactionByHash(address: Encoded.AccountAddress, hash: string) {
+  return transactions.value[address]?.loaded.find((transaction) => transaction.hash === hash);
+}
+
+function updateAccountTransactions(address: Encoded.AccountAddress, transaction: ITransaction) {
+  const transactionIndex = transactions.value[address]?.loaded?.findIndex(
+    (tr) => tr.hash === transaction.hash,
+  );
+
+  if (transactionIndex >= 0) {
+    transactions.value[address].loaded[transactionIndex] = transaction;
+  }
+}
+
+const setTransactionsNextPage = (address: Encoded.AccountAddress, url: string) => {
+  transactions.value[address].nextPageUrl = url;
+};
+
+export function useTransactionList() {
+  const store = useStore();
+  const { nodeNetworkId, getAeSdk } = useAeSdk({ store });
+  const { activeAccount, isLoggedIn } = useAccounts({ store });
+
+  const allTransactionsForAccount = computed(() => {
+    if (!isLoggedIn) {
+      return [];
+    }
+    const { pending, loaded } = getOrInitTransactionStateForAccount(activeAccount.value.address);
+
+    return [...loaded, ...(pending[nodeNetworkId.value!] || [])];
+  });
+
+  function setPendingTransactionSentByHash(address: Encoded.AccountAddress, hash: string) {
+    const index = transactions.value[address].pending[nodeNetworkId.value!]
+      ?.findIndex((transaction) => transaction.hash === hash);
+    if (index >= 0) {
+      transactions.value[address]
+        .pending[nodeNetworkId.value!][index][TX_DIRECTION.sent] = true;
+    }
+  }
+
+  async function waitTransactionMined(address: Encoded.AccountAddress, hash: string) {
+    try {
+      const aeSdk = await getAeSdk();
+      await aeSdk.poll(hash as Encoded.TxHash);
+      setPendingTransactionSentByHash(address, hash);
+    } catch (e) {
+      store.commit('removePendingTransactionByHash', { hash, network: nodeNetworkId.value });
```

Comment:
Maybe you have placed this code by mistake.  Pending transactions have been removed from the store as I see.